dna/examine_test.go

```diff
@@ -38,6 +38,12 @@ func TestCountGaps(t *testing.T) {
 	}
 }
 
+func TestGCContent(t *testing.T) {
+	if GCContent(alpha) != 50 {
+		t.Errorf("problem with GCContent")
```

Comment:
Small style suggestion/preference to stay consistent with repository conventions
```suggestion
		t.Errorf("Error: Problem with GCContent")
```